### PR TITLE
Problem: need ability to add new cell to notebook

### PR DESCRIPTION
**Pull request by rgbkrk**
Solution: provide action and reduction for cell creation

### CODE REVIEW

File: src/components/cell/code-cell.js [hunk 1]
```diff
@@ -25,10 +26,23 @@ export default class CodeCell extends React.Component {
...
+    if(!shiftXORctrl) {
+      return;
+    }
+
+    if (e.shiftKey) {
+      // TODO: Remove this, as it should be created if at the end of document only
+      // this.context.dispatch(createCellAfter('code', this.props.id));
+
+      // should instead be
+      // this.context.dispatch(nextCell(this.props.id));
```
Comment (rgbkrk) [inline src/components/cell/code-cell.js:43]: Of note, this doesn't actually exist yet... I just wanted to outline how it could be used.

**Comment by ivanov** [comment]
Problem: all commit messages start with Problem:
;)

**Comment by rgbkrk** [comment]
Problem: You know, it's grinding on me too.

Solution: Stop using it

/cc @jdfreder

**Comment by captainsafia** [comment]
I got 99 problems....

**Comment by ivanov** [comment]
but misogynistic pop songs aren't one?

On Tue, Feb 16, 2016 at 9:41 PM, Safia Abdalla notifications@github.com
wrote:

> I got 99 problems....
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nteract/nteract/pull/165#issuecomment-185036547.

## 

Paul Ivanov
http://pirsquared.org | GPG/PGP key id: 0x0F3E28F7

**Pull request merged**